From 3cb861a48819becb8f3f5c66d3aa62e6a5fd620f Mon Sep 17 00:00:00 2001
From: Jes Sorensen <Jes.Sorensen@redhat.com>
Date: Mon, 1 Sep 2014 09:25:08 -0400
Subject: [md] Fix apparent cut-and-paste error in super_90_validate

Message-id: <1409563584-9000-7-git-send-email-Jes.Sorensen@redhat.com>
Patchwork-id: 89941
O-Subject: [RHEL7.1 PATCH 06/82] md: Fix apparent cut-and-paste error in super_90_validate
Bugzilla: 1085530
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

From: Dave Jones <davej@redhat.com>

Setting a variable to itself probably wasn't the intention here.

Signed-off-by: Dave Jones <davej@fedoraproject.org>
Signed-off-by: NeilBrown <neilb@suse.de>
(cherry picked from commit c9ad020fec895bf1e5fcc322d0ab9e67efd3e3a0)
Signed-off-by: Jes Sorensen <Jes.Sorensen@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/md/md.c b/drivers/md/md.c
index 6b2ae33..7ee2b6c 100644
--- a/drivers/md/md.c
+++ b/drivers/md/md.c
@@ -1180,7 +1180,7 @@ static int super_90_validate(struct mddev *mddev, struct md_rdev *rdev)
    mddev->bitmap_info.offset =
     mddev->bitmap_info.default_offset;
    mddev->bitmap_info.space =
-    mddev->bitmap_info.space;
+    mddev->bitmap_info.default_space;
   }
 
  } else if (mddev->pers == NULL) {
-- 
1.7.1